### ISSUE TITLE
Could not use "Codedungeon\\PHPUnitPrettyResultPrinter\\Printer" as printer: class does not exist

### ISSUE THREAD

**Issue by okaufmann** [mikeerickson/phpunit-pretty-result-printer#13]
Hi,

I tried to use the Printer on windows with phpunit 6.5.5 in a Laravel Project.

When I add it to phpunit.xml and run the tests they appear to run forever with no output.
When I set the printer in commandline I get the following error:

```
$ phpunit --printer=Codedungeon\\PHPUnitPrettyResultPrinter\\Printer
PHPUnit 6.5.5 by Sebastian Bergmann and contributors.

Could not use "Codedungeon\\PHPUnitPrettyResultPrinter\\Printer" as printer: class does not exist
```

Is this a Windows related issue or am a doing something wrong?

Thanks

**Comment by guruas3** [comment]
I have the same issue on Windows.

The problem is in this block of code: [code](https://github.com/mikeerickson/phpunit-pretty-result-printer/blob/master/src/Printer.php#L267-L273) as it never closes the while loop.
Because this condition : [code](https://github.com/mikeerickson/phpunit-pretty-result-printer/blob/master/src/Printer.php#L269) is never satisfied on Windows.
As `$configPath` changes in the while loop and goes 1 directory up at a time it reaches this value `C:\` and then goes into infinite loop.

I'm not sure about the ideal solution but i've temporarily done this for myself locally: 
`$configPath === "/" || (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') && $configPath === 'C:\\'`

P.S. the tests written for Printer class also going into infinite loop.

**Comment by mikeerickson** [comment]
Hey guys, I am on vacation until Tuesday. I will have a look at this when I return.

**Comment by mfour-merickson** [comment]
@okaufmann @guruas3 Hey guys, just getting back to these issues.  Can you please share with me what version of Windows you are using.

**Comment by guruas3** [comment]
I have Windows 7. But i don't think it's relevant as all versions of Windows should have the same error. I showed you why in my previous comment. `$configPath` will be equal to `/` just in linux systems, and in windows it will be in the different format `C:\`

**Comment by mikeerickson** [comment]
I am going to be testing with Windows 10 and will update accordingly.  Just wanted to make sure to cover all the possible basis

**Comment by mikeerickson** [comment]
@guruas3 On the surface, it looks like you adjustments will be satisfactory, but will do some more thorough testing to make sure.

**Comment by guruas3** [comment]
@mikeerickson Yeah sure. The problem with my fix is that it depends on the disk where the project is located. So let's suppose i have a project in `D:\MyProjects\etc` the `$configPath` will become `D:\` and not `C:\`. The only thing that comes to my mind is refactor the `while` loop.

**Comment by mikeerickson** [comment]
I have a solution for this issue, just need to do more testing and adding some other changes planed for next version.

**Comment by okaufmann** [comment]
I'm using it on Windows 10

**Comment by EmanueleCoppola** [comment]
@mikeerickson any news on this?

**Comment by Qussayyon** [comment]
@mikeerickson 
I'm using PHP 7.2, on Ubuntu 18.04.
Still giving me error:
`Could not use "Codedungeon\PHPUnitPrettyResultPrinter\Printer" as printer`

**Comment by peterlupu** [comment]
for those on windows, see #39

**Comment by mikeerickson** [comment]
As commented on #39 I will be merging and tagging this afternoon.

**Comment by mikeerickson** [comment]
Fixed with #39
